OstelcoStyles/Info.plist

```diff
@@ -17,6 +17,6 @@
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>41</string>
```

Comment:
@designatednerd  We use increment_build_number in fastlane to bump the version number, but seems it's overwriting some other logic you have added

Reply:
This is fine, we want that to get updated by `increment_build_number`

Reply:
great thanks!